src/lib/operators/table_scan/column_vs_value_table_scan_impl.cpp

```diff
@@ -100,7 +100,8 @@ void ColumnVsValueTableScanImpl::_scan_segment(const BaseDictionarySegment& segm
   if (_value_matches_all(segment, search_value_id)) {
     iterable.with_iterators(position_filter, [&](auto it, auto end) {
       static const auto always_true = [](const auto&) { return true; };
-      _scan_with_iterators<false>(always_true, it, end, chunk_id, matches);
+      // MatchesAll, so include all rows except those with NULLs in the result.
```

Comment:
Where does "MatchesAll" come from? It looks like an enum or a class.

Reply:
My personal whimsy of camel casing every word that I want to stand out....